public/app/features/live/pages/RuleTest.tsx

```diff
@@ -30,7 +30,7 @@ export const RuleTest: React.FC<Props> = (props) => {
         if (t) {
           setResponse(
             t.map((f) => {
-              const frame = new StreamingDataFrame(f.frame);
+              const frame = StreamingDataFrame.fromDataFrameJSON(f.frame);
```

Comment:
```suggestion
              const frame = dataFrameFromJSON(f.frame);
```
No need for the streaming flavor here